console/ui/src/app/console.service.ts

```diff
@@ -1183,11 +1183,11 @@ export class ConsoleService {
   }
 
   /** List (and optionally filter) storage data. */
-  listStorage(auth_token: string, userId?: string, key?: string, collection?: string, cursor?: string): Observable<StorageList> {
+  listStorage(auth_token: string, user_id?: string, key?: string, collection?: string, cursor?: string): Observable<StorageList> {
 		const urlPath = `/v2/console/storage`;
     let params = new HttpParams();
-    if (userId) {
-      params = params.set('userId', userId);
+    if (user_id) {
```

Comment:
just to confirm; the changes done here (which are what we agreed on our call); do they affect the StorageList object, and therefore what the dashboard TypeScript code relies on? if not, then this PR LGTM.

Reply:
Nope, these query params are passed as function args, so nothing changes for the caller:

`listStorage(auth_token: string, user_id?: string, key?: string, collection?: string, cursor?: string)`